client/src/components/Login/LoginContainer.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { LoginContainer } from './LoginContainer';
+import * as loginuser from '../../store/login/actions';
+import * as settoken from '../../store/login/setToken.helper';
+import * as reactstrap from 'reactstrap';
+
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import React from 'react';
+import { Provider } from 'react-redux';
+
+describe('LoginContainer component:', () => {
+    let renderer;
+
+    beforeEach(() => {
+        const handleSubmit = jest.fn(() => 'somehandlesubmit');
+
+        loginuser.loginUser = jest.fn(() => 'somevalue');
+        reactstrap.Form = jest.fn(() => (<div></div>));
+        settoken.setToken = jest.fn(() => 'somevalue');
+
+        const match = { params: { token: 'sometoken' } };
+
+        // tslint:disable:trailing-comma
+        const store = configureStore()();
+        renderer = shallow(
+            <Provider store={store}>
+                <LoginContainer match={match} handleSubmit={handleSubmit} />
+            </Provider>
+        );
+        // tslint:enable:trailing-comma
```

Comment:
too much this rules. Can we remove trailing-comma or change linter?

Reply:
Refactored.
I actually like trailing commas in multiline function arguments; this issue assessment sounds like a bug (probably tsx-related). Unfortunately tslint is on track to be merged with eslint and I'm not sure a) whether eslint works well with TypeScript b) whether tslint community would accept bug report, take apart act on it swiftly. Thus quick (though not very pretty) workaround seems to be an optimal solution.

Reply:
I like trailing commas too. Can we change config?

Reply:
I can't see a simple solution to this proble. Tslint (falsely) identifies
```
return (
    <ReactElement />
);
```
as multiline function call and asks for a trailing comma after `/>`. Options are:
- comply (which breaks every style guide I could think of)
- disable block with `// tslint:disable...` (original solution)
- trick tslint with parenthesis coming right after `/>` (current solution)
- disable trailing commas for multiline function calls (approx. 20-30 changes in codebase)
- go for an (unlikely) bugfix in tslint
- attempt to switch to eslint (with unpredictable result)